core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -475,7 +475,52 @@ class DAGSchedulerSuite
     assert(results === Map(0 -> 42, 1 -> 43))
     assertDataStructuresEmpty()
   }
+  
+  test("Test taskAbort after multiple stage failures.") {
```

Comment:
Kay - could you please point me at an example of a stage with multiple tasks in one of the tests just to help provide a baseline? There's no examples of this within the DAGSchedulerSuite and I'm having a bit of trouble figuring out how to code this up.